### DISCUSSION TITLE
When a sidecar is injected into cert-manager pod, certificate is not issued.

### DISCUSSION THREAD

**Discussion by jjangga0214** [linkerd/linkerd2#11457] [General] [answered]
When a sidecar is injected into cert-manager pod, certificate is not issued.

If I disable the injection, it works.

Is this a known issue?

Thanks.

**Comment by alpeb** [comment, marked as answer]
We disable [by defaullt](https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/values.yaml#L371) the injection of cert-manager, because the linkerd-proxy injector might race with cert-manager's ca-injector. See #7699 .

**Reply by jjangga0214** [reply]
Thanks! Is this written in the official docs? I haven't read about it. It might be worth documenting. Because so many people use cert-manager that someone may try injecting a sidecar. The source code seems to check namespace `cert-manager` but actually it can be on another namespace. How do you think?

**Reply by alpeb** [reply]
I couldn't find anything about it in the docs, so good point :+1: If you're up to it, we'd love to review a PR in the [website](https://github.com/linkerd/website) repo to address this :wink:
